README.md

```diff
@@ -34,6 +34,7 @@ We'll organise into different streams here. One kind request for motivation: Ple
 * Restaurant (Landing Page)
 * Quiz Game
 * Podcast Website
+*Landing page
```

Comment:
1. All these are landing pages. Please specify which type of landing page.
2. Please add a space after the `*` so that it is considered as list item.
3. Can you please make all the Pull Requests in a single PR as this might violate the rules of Hacktoberfest.